src/stim/simulators/force_streaming.h

```diff
@@ -0,0 +1,11 @@
+#include <cstdint>
+
+namespace stim {
+
+bool should_use_streaming_because_bit_count_is_too_large_to_store(uint64_t result_count);
```

Comment:
While we're here, add a comment saying that this is here for testing the streaming without having to make enormous circuits.